### PR TITLE
v7.0 Docs Update

### PR DESCRIPTION
**Pull request by coliff**
- Remove unneeded x-ua-compatible - not needed for IE11
- Update many external links
- Fix typos
- Add info for creating a good meta description
- Add link for iPhone X viewport info
- Update browser upgrade prompt
- Remove integrating Bootstrap with H5BP (it was written a longtime ago and I don't think is relevant so much)
- Remove statement that Twitter Cards requires registering a domain (its now open for all sites)
- Remove official shortlink section, it was always very poorly supported
- Made the touch icons sections more concise (no need to mention about non-retina iPhones and iOS versions prior to iOS 6 etc)
- Add `<meta name="theme-color" content="#fafafa">` to `index.html`
- Add details on `security.txt` in extend docs

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

### CODE REVIEW

**Comment by Suneet-M** [comment]
Hey, you need any help? Want to get some open source experience, pretty new here.

File: dist/doc/extend.md [hunk 1]
```diff
@@ -620,19 +586,14 @@ Icons](https://mathiasbynens.be/notes/touch-icons).
 
 Apart from that it is possible to add start-up screens for web apps on iOS. This
 basically works by defining `apple-touch-startup-image` with an according link
-to the image. Since iOS devices have different screen resolutions it is
+to the image. Since iOS devices have different screen resolutions it maybe
```
Comment (roblarsen) [inline dist/doc/extend.md]: may be

File: src/doc/extend.md [hunk 2]
```diff
@@ -620,19 +586,14 @@ Icons](https://mathiasbynens.be/notes/touch-icons).
 
 Apart from that it is possible to add start-up screens for web apps on iOS. This
 basically works by defining `apple-touch-startup-image` with an according link
-to the image. Since iOS devices have different screen resolutions it is
+to the image. Since iOS devices have different screen resolutions it maybe
```
Comment (roblarsen) [inline src/doc/extend.md]: where we actually need to make the change

**Comment by roblarsen** [comment]
going once.. going twice..

**Comment by afino** [comment]
Just wanted to point out that there is still a link/reference to integrating Bootstrap with H5BP in the FAQ, but there is no such content. Consider either re-adding the content or removing the link.

**Pull request merged**